src/components/facet-header.tsx

```diff
@@ -0,0 +1,88 @@
+import '@isrd-isi-edu/chaise/src/assets/scss/_faceting.scss';
+
+import { useRef, useState } from 'react';
+
+// Components
+import OverlayTrigger from 'react-bootstrap/OverlayTrigger';
+import Tooltip from 'react-bootstrap/Tooltip';
+import DisplayValue from '@isrd-isi-edu/chaise/src/components/display-value';
+import { Displayname } from '@isrd-isi-edu/chaise/src/models/displayname';
+
+type FacetHeaderProps = {
+  displayname: Displayname; // content to be displayed as panel header
+  showTooltipIcon?: boolean; // prop to enable tooltip for displayname
+  comment?: string; // Optional text to be shown on hover of displayname
+};
+
+/**
+ * @returns FacetHeader Component
```

Comment:
Just add a line explaining what this is.. I know the name is self explanatory, but for completeness let's add it.